src/odemis/gui/util/img.py

```diff
@@ -1904,7 +1916,10 @@ def _adapt_rgb_to_raw(imrgb, stream, dtype):
     return (ndarray Y,X)
     """
 
-    data_raw = stream.raw[0]
+    if isinstance(stream.raw, list):
+        data_raw = stream.raw[0]
+    else:
```

Comment:
Again, first read data_raw, then if it's a 2D tuple, use mergeTiles(). (Actually, here we use data_raw just for min/max, so calling mergeTiles() is overkill, so you could either do directly compute the min/max on the tiles, or just add a comment that it could be optimised.